crypto/hmac/build.info

```diff
@@ -1,3 +1,6 @@
 LIBS=../../libcrypto
 SOURCE[../../libcrypto]=\
         hmac.c hm_ameth.c hm_meth.c
+
+SOURCE[../../providers/fips]=\
+        hmac.c hm_meth.c
```

Comment:
May I suggest this?

```
$COMMON=hmac.c hm_meth.c

LIBS=../../libcrypto
SOURCE[../../libcrypto]=$COMMON hm_ameth.c
SOURCE[../../providers/fips]=$COMMON
```

Reply:
Ah, yes. This code was written before you introduced that capability. Fixed.